### PR TITLE
Improved Git-Github Roadmap

### PR DESCRIPTION
**Pull request by offensive-vk**
## Improvements

- Add Missing Anchors.
- Add Learning Resources.
- Fix Word or Spelling Mistakes.
- Add Missing Content.
- Fix Markdown Formatting.
- Fix Wrongly Typed `@type@`
- Add Official Links for Learning.
- Add Missing Links Sections.

---

Thankyou.

### CODE REVIEW

**Comment by offensive-vk** [comment]
@dansholds , Another Huge PR Mate 😂

**Comment by dansholds** [comment]
> @dansholds , Another Huge PR Mate 😂

Why though, you said you would submit in smaller PRs moving forwards.

**Comment by offensive-vk** [comment]
> > @dansholds , Another Huge PR Mate 😂
> 
> Why though, you said you would submit in smaller PRs moving forwards.

Its very time consuming to open a pr for every 5 topics lol

**Comment by offensive-vk** [comment]
Bro why is there a collaborator badge under yo name, whereas contributor under mine 🤨

**Comment by dansholds** [comment]
> Bro why is there a collaborator badge under yo name, whereas contributor under mine 🤨

You're a contributor, I am a collaborator, I'm a full time employee of Insight Media Group and Roadmap.sh.

File: src/data/roadmaps/git-github/content/creating-account@i7fIIHcBEk473te8bniJ5.md [hunk 1]
```diff
@@ -4,4 +4,5 @@ To get started with GitHub, you'll need to create a free personal account on Git
 
 Visit the following resources to learn more:
 
-- [@official@Creating an account on GitHub](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github)
+- [@official@Creating an Account on GitHub](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github)
+- [@official@Your Github Account](https://github.com/login)
```
Comment (dansholds) [inline src/data/roadmaps/git-github/content/creating-account@i7fIIHcBEk473te8bniJ5.md:8]:
```suggestion
```

Comment (dansholds) [reply src/data/roadmaps/git-github/content/creating-account@i7fIIHcBEk473te8bniJ5.md:8]: linking to the login page isn't needed.

File: src/data/roadmaps/git-github/content/profile-readme@KbVvFVQ4k77R2_ddsc7WT.md [hunk 2]
```diff
@@ -2,8 +2,7 @@
 
 A GitHub Profile README is a special repository that allows users to showcase their skills, projects, and personality directly on their GitHub profile. To create one, you need to make a new repository with the same name as your GitHub username. This repository should contain a README.md file, which GitHub will automatically display on your profile page. The README can be customized with Markdown formatting, allowing you to add text, images, links, and even dynamic content like GitHub stats or recent blog posts. This feature provides a unique opportunity to make your GitHub profile more engaging and informative for visitors, effectively serving as a personalized landing page for your GitHub presence.
 
-
 Visit the following resources to learn more:
 
-- [@official@Managing your profile README](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)
+- [@official@Managing your Profile Readme](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)
```
Comment (dansholds) [inline src/data/roadmaps/git-github/content/profile-readme@KbVvFVQ4k77R2_ddsc7WT.md:7]:
```suggestion
- [@official@Managing your Profile README](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)
```

Comment (dansholds) [reply src/data/roadmaps/git-github/content/profile-readme@KbVvFVQ4k77R2_ddsc7WT.md:7]: README is all caps.

File: src/data/roadmaps/git-github/content/roadmaps@sxvT2hGko2PDRBoBrCGWD.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Roadmaps
+# Roadmap(s)
```
Comment (dansholds) [inline src/data/roadmaps/git-github/content/roadmaps@sxvT2hGko2PDRBoBrCGWD.md:1]:
```suggestion
# Roadmaps
```

File: src/data/roadmaps/git-github/content/roadmaps@sxvT2hGko2PDRBoBrCGWD.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
-GitHub roadmaps are a feature that helps you visualize and organize plans for your projects, allowing you to create a high-level view of milestones and goals, and collaborate on planning and tracking progress with team members.
+GitHub Roadmap(s) are a feature that helps you visualize and organize plans for your projects, allowing you to create a high-level view of milestones and goals, and collaborate on planning and tracking progress with team members.
```
Comment (dansholds) [inline src/data/roadmaps/git-github/content/roadmaps@sxvT2hGko2PDRBoBrCGWD.md:3]:
```suggestion
GitHub Roadmaps are a feature that helps you visualize and organize plans for your projects, allowing you to create a high-level view of milestones and goals, and collaborate on planning and tracking progress with team members.
```

**Comment by offensive-vk** [comment]
Great 👏🏻

Thanks for clarifying boss.

**Review by dansholds — APPROVED** [review]

**Comment by dansholds** [comment]
LGTM, I get that it's "annoying" to open a PR for 5 or so files at a time but that's good practice so let's try that next time.

FYI this PR was a lot better than the last one, good job.

**Comment by offensive-vk** [comment]
Sure thing 😊

**Comment by kamranahmedse** [comment]
@dansholds actually let's do single PRs for content changes like this (functionality should be multiple small PRs, I agree).

@offensive-vk content changes like this are fine in a single PR.

**Comment by offensive-vk** [comment]
Couldn't agree more sir.
Noted

**Pull request merged**